dmscripts/upload_counterpart_agreements.py

```diff
@@ -18,6 +18,7 @@ def upload_counterpart_file(
     file_path,
     dry_run,
     data_api_client,
+    contract_title,
```

Comment:
Rather than passing just the contact title why not pass the content loader object? I think that would make it easier if we need more stuff from the content loader later on.

Reply:
@lfdebrux I thought about this but it is inefficient as we then have to re-load this value for every file in the batch. Also simpler this way if we decide the value should live somewhere other than content loader in the future.

Reply:
Okay, makes sense.